packages/calcite-components/src/components/input-date-picker/input-date-picker.scss

```diff
@@ -1,55 +1,114 @@
+ * @prop --calcite-input-date-picker-calendar-shadow: the component's calendar shadow.
+ * 
+ * @prop --calcite-input-date-picker-icon-color: Specifies the component's in-field input icon color.
+ * @prop --calcite-input-date-picker-icon-color-hover: Specifies the component's in-field input icon color on hover.
+ * @prop --calcite-input-date-picker-actions-icon-color: Specifies the component's actions icon color.
+ * @prop --calcite-input-date-picker-actions-icon-color-hover: Specifies the component's actions icon color on hover.
+ * @prop --calcite-input-date-picker-calendar-icon-color: Specifies the component's calendar icon color.
+ * @prop --calcite-input-date-picker-calendar-icon-color-hover: Specifies the component's calendar icon color.
+ * 
+ * @prop --calcite-input-date-picker-background-color: Specifies the component's input background color.
+ * @prop --calcite-input-date-picker-calendar-actions-background-color: Specifies the background color of actions of the calendar.
+ * @prop --calcite-input-date-picker-calendar-actions-background-color-hover: Specifies the background color of actions of the calendar when hovered.
+ * @prop --calcite-input-date-picker-calendar-actions-background-color-press: Specifies the background color of actions of the calendar when pressed.
+ * @prop --calcite-input-date-picker-calendar-selected-background-color: Specifies the background color of selected day and day range of the calendar.
+ * @prop --calcite-input-calendar-day-background-color: Specifies the background color of day of the calendar.
+ * @prop --calcite-input-date-picker-calendar-day-background-color-hover: Specifies the background color of day of the calendar when hovered.
+ * @prop --calcite-input-date-picker-calendar-day-outside-range-background-color-hover: Specifies the background color of day's outside current range of the component's date-picker when hovered.
+ * 
+ * @prop --calcite-input-date-picker-corner-radius: Specifies the component's input corner radius.
+ * @prop --calcite-input-date-picker-calendar-corner-radius: Specifies the component's calendar border radius.
+ *
+ * @prop --calcite-input-date-picker-text-color: Specifies the component's input text color.
+ * @prop --calcite-input-date-picker-placeholder-text-color: Specifies the component's input placeholder text color.
+calendars when `range="true"`.
+ * @prop --calcite-input-date-picker-calendar-actions-text-color: Specifies the text color of the component's calendar actions.
+ * @prop --calcite-input-date-picker-calendar-actions-text-color-press: Specifies the text color of the component's calendar actions when pressed.
+ * @prop --calcite-input-date-picker-calendar-text-color: Specifies the text color of the component's calendar week, year & suffix.
+ * @prop --calcite-input-date-picker-calendar-month-select-font-size: Specifies the font size of month select of the component's date-picker.
```

Comment:
I don't know if we need to expose this font size css property cc @ashetland / @SkyeSeitz